content/en/docs/setup/version-skew-policy.md

```diff
@@ -0,0 +1,128 @@
+---
+reviewers:
+- sig-api-machinery
+- sig-architecture
+- sig-cli
+- sig-cluster-lifecycle
+- sig-node
+- sig-release
+title: Kubernetes Version and Version Skew Support Policy
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+This document describes the maximum version skew supported between various Kubernetes components.
+Specific cluster deployment tools may place additional restrictions on version skew.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Supported versions
+
+Kubernetes versions are expressed as **x.y.z**,
+where **x** is the major version, **y** is the minor version, and **z** is the patch version, following [Semantic Versioning](http://semver.org/) terminology.
+For more information, see [Kubernetes Release Versioning](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#kubernetes-release-versioning).
+
+The Kubernetes project maintains release branches for the most recent three minor releases,
+and expects users to be running those versions in production.
+
+Applicable bug and security fixes may be backported to those three release branches, depending on severity and feasibility.
+Patch releases are cut from those branches at a regular cadence,
+or on an as-needed basis as determined by the [patch release manager](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/patch-release-manager/README.md#release-timing).
+The patch release manager is listed in the [release team for each release](https://github.com/kubernetes/sig-release/tree/master/releases/).
+
+With minor releases happening approximately every three months,
+that means each minor release is maintained for approximately nine months.
+
+## Supported version skew
+
+### `kube-apiserver`
+
+In multi-server clusters, the newest and oldest `kube-apiserver` instances must be within one minor version.
+
+Example:
+
+* newest `kube-apiserver` is at **1.13**
+* other `kube-apiserver` instances are supported at **1.13** and **1.12**
+
+### `kubelet`
+
+`kubelet` must not be newer than `kube-apiserver`, and may be up to two minor versions older.
+
+Example:
+
+* `kube-apiserver` is at **1.13**
+* `kubelet` is supported at **1.13**, **1.12**, and **1.11**
+
+Note that if version skew exists between `kube-apiserver` instances in a multi-server cluster, this narrows the allowed `kubelet` versions.
+
+Example:
+
+* `kube-apiserver` instances are at **1.13** and **1.12**
+* `kubelet` is supported at **1.12**, and **1.11** (**1.13** is not supported because that would be newer than the `kube-apiserver` instance at version **1.12**)
+
+### `kube-controller-manager` and `kube-scheduler`
```

Comment:
Usage of `cloud-controller-manager` is relatively small (at least from what I can tell) but I think it's still worth calling out here because some providers _have_ to use it. I think just having a side note of "upgrade along with kube-controller-manager" is sufficient.

Reply:
Happy to add this section myself once this PR is merged :)

Reply:
added